### PR TITLE
OCPBUGS-17558_RN:Add known issue to 4.12 RNs for Azure File NFS support

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.12 only

Issue: https://issues.redhat.com/browse/OCPBUGS-17558

Link to docs preview: https://65713--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes#ocp-4-12-known-issues

QE review:
- [x] QE has approved this change.

Additional information:

PR to add same info to Azure File CSI driver content: https://github.com/openshift/openshift-docs/pull/63854

**PTAL**: @bertinatto, @duanwei33, @gcharot, @jsafrane, @sferich888, @xltian, @mwerner2113

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -3244,6 +3244,10 @@ There is currently no workaround to resolve this issue if encountered. To avoid
 
 * Currently, when using a persistent volume (PV) that contains a very large number of files, the pod might not start or can take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
 
+* Creating pods with Azure File NFS volumes that are scheduled to the control plane node causes the mount to be denied. (link:https://issues.redhat.com/browse/OCPBUGS-18581[*OCPBUGS-18581*])
++
+To work around this issue: If your control plane nodes are schedulable, and the pods can run on workers nodes, use nodeSelector or Affinity to schedule the pod in worker nodes.
```
Comment (skopacz1) [inline release_notes/ocp-4-12-release-notes.adoc:3249]: Should nodeSelector be in backticks?

```suggestion
To work around this issue: If your control plane nodes are schedulable, and the pods can run on workers nodes, use `nodeSelector` or Affinity to schedule the pod in worker nodes.
```

**Review by skopacz1 — COMMENTED** [review]
Left one comment, otherwise looks good!

**Comment by gcharot** [comment]
LGTM

**Comment by bertinatto** [comment]
lgtm

**Comment by mwerner2113** [comment]
LGTM

**Comment by xltian** [comment]
lgtm

**Pull request merged**